Cards/MadGraph5_aMCatNLO/DY/DYto2L-2Jets_MLL-50_PTLL-600_2J_amcatnloFXFX-pythia8/DYto2L-2Jets_MLL-50_PTLL-600_2J_amcatnloFXFX-pythia8_proc_card.dat

```diff
@@ -0,0 +1,8 @@
+import model loop_sm-ckm_no_b_mass
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+generate p p > ell+ ell- j [QCD] @0
```

Comment:
Maybe this is related to the FYI. Can you elaborate on that? I see this is DYto2L-2Jets --> but only 1 jet here. 

I thought the names was to identify:
DYto2L-2Jets_*_1J_* --> DYto2L+2Jets; this is the 1J bin
DYto2L-2Jets_*_2J_* --> DYto2L+2Jets; this is the 2J bin

Can you confirm?

Reply:
Right: DYto2L-2Jets (is the full process), _1J is the "jet selection cut" of that sample. so just
p p > e+ e- j [QCD]
The point is that we want to clarify for users the global setup (2 Jets) and make sure that the sample is properly combined with 0 and 2 jets to get the full description.

Reply:
@Cvico ah it's fixed in the master branch